hie.yaml

```diff
@@ -0,0 +1,7 @@
+cradle:
+  cabal:
+    - path: "./src"
+      component: "lib:apropos"
+
+    - path: "./examples"
+      component: "apropos:test:examples"
```

Comment:
I'm pretty sure lines 3-7 are redundant,  at least if these packages are listed [`cabal.project`](https://github.com/mlabs-haskell/apropos/blob/main/cabal.project#L3).

Reply:
I didn't know this. Is there any reason to prefer a blank `hie.yaml`? I have added `implicit-hie`, so we don't even really have to think hard about updating it.

Reply:
Is `implicit-hie` actually required? I thought having `tools.haskell-language-server = { };` and that 2-line bare hie.yaml should suffix.

Reply:
cf. 
- https://github.com/Plutonomicon/plutarch/blob/staging/hie.yaml
- https://github.com/Plutonomicon/plutarch/blob/9a173e605fa0bbd66010e33e79e376602d0a8b6d/cabal.project#L1-L3